helpers/errorHandlers.js

```diff
@@ -1,32 +1,38 @@
 const logger = require('winston');
 const { postToStats } = require('../controllers/statsController.js');
 
+function useObjectPropertiesIfNoKeys (err) {
+  if (Object.keys(err).length === 0) {
+    let newErrorObject = {};
+    Object.getOwnPropertyNames(err).forEach((key) => {
```

Comment:
When is there a non-enumerable property being passed that this is required?

Reply:
for errors created by node (e.g. `new Error('error message')`) the stack and message are non-enumerable